examples/github.com-dakanji-RefindPlus/build.sh

```diff
@@ -3,6 +3,10 @@
 mkdir -m 1777 out
 
 git clone --recursive https://github.com/dakanji/RefindPlus RefindPlusPkg
+chmod -R 1777 RefindPlusPkg
+# Comment out the docker pull line to use the rebuild Docker image in Makefile 
+# Remove the comments in the Makefile as well to start the rebuild process.
+
 docker pull xaionaro2/edk2-builder:RefindPlusUDK
```

Comment:
We actually can just remove `docker pull` for all cases. If `docker build` finds no local image, then it invokes `docker pull` anyway.

Reply:
done